packages/cubejs-api-gateway/src/gateway.ts

```diff
@@ -339,29 +329,20 @@ class ApiGateway {
     // Used by Rollup Designer
     app.post(
       `${this.basePath}/v1/pre-aggregations/can-use`,
-      [
-        ...userMiddlewares,
-        async (req, res, next) => {
-          try {
-            await this.assertApiScope(
-              'meta',
-              req?.context?.securityContext
-            );
-            if (next) next();
-          } catch (e: unknown) {
-            if (e instanceof CubejsHandlerError) {
```

Comment:
There is no call for `next` when error != `CubejsHandlerError`.
This will lead to unhandled connection.